rmw_opensplice_cpp/CMakeLists.txt

```diff
@@ -28,12 +28,14 @@ if(NOT rosidl_typesupport_opensplice_cpp_FOUND OR NOT rosidl_typesupport_openspl
   return()
 endif()
 
+find_package(rcpputils REQUIRED)
 find_package(rcutils REQUIRED)
 find_package(rmw REQUIRED)
 find_package(rosidl_generator_c REQUIRED)
 find_package(rosidl_generator_cpp REQUIRED)
 
 ament_export_dependencies(
+  rcpputils
```

Comment:
@jacobperron The package exports the `rcpputils` dependency but doesn't declare that run dependency in the manifest.

Reply:
Addressed in https://github.com/ros2/rmw_opensplice/pull/277